wooey/backend/utils.py

```diff
@@ -422,16 +440,16 @@ def test_fastx(filepath):
 
 def create_job_fileinfo(job):
     parameters = job.get_parameters()
-    from ..models import WooeyFile
+    from ..models import WooeyFile, UserFile
```

Comment:
Everywhere else `WooeyFile` is replaced with `UserFile`, but here both are imported. Is this correct?